soroban-sdk/src/tests/contract_docs_enum_int.rs

```diff
@@ -0,0 +1,38 @@
+use crate as soroban_sdk;
+use soroban_sdk::contracttype;
+use stellar_xdr::{ReadXdr, ScSpecEntry, ScSpecUdtEnumCaseV0, ScSpecUdtEnumV0};
+
+/// E has variants A and B.
+#[contracttype]
+#[derive(Copy, Clone)]
+pub enum E {
+    /// A is a.
```

Comment:
Can we cover a multi-line comment for enum variant somewhere?

Reply:
Added some multiline cases in fce2c958.